roles/kubernetes-apps/network_plugin/kube-router/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: kube-router | Start Resources
+  kube:
+    name: "kube-router"
+    kubectl: "{{ bin_dir }}/kubectl"
+    filename: "{{ kube_config_dir }}/kube-router.yml"
+    resource: "ds"
+    namespace: "kube-system"
+    state: "latest"
+  when: inventory_hostname == groups['kube-master'][0]
+
+- name: kube-router | Wait for kube-router pods to be ready
+  command: "{{bin_dir}}/kubectl -n kube-system get pods -l k8s-app=kube-router -o jsonpath='{.items[?(@.status.containerStatuses[0].ready==false)].metadata.name}'"
```

Comment:
Any reason not to use the standard labels from https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/ ?

Reply:
Couple reasons:
- compatibility with upstream kube-router manifests (which use those labels)
- had already found some at the kubespray repo (`egrep -r k8s-app:`)

If you feel strong about it, ofc I can change it anyways.